### ISSUE TITLE
bug: Faugra 0.0.45 (OSX?)

### ISSUE THREAD

**Issue by daniel-hritcu** [zvictor/faugra#25]
0.0.44 installs fine, but in 0.0.45 when trying to do
` pnpm i faugra` or `npm i -g faungra`, it fails to execute the postinstall script with

```bash
command sh -c ./scripts/postinstall.js
sh: ./scripts/postinstall.js: Permission denied
```

In order to fix I had to make the file executable in order to run.

Another thing is that when the duplicated `locateCache` entry was removed from `utils.js`, the new file, `locateCache.cjs` was not exported in the package build so it's missing, throwing a file not found error when running post install.

Environment:
node 18.0
npm 8.6.0
pnpm 7.1.5
OS: Mac Os 13.0

**Comment by zvictor** [comment]
Thank you for reporting it @daniel-hritcu !
I will do my best to publish a new version still this weekend.

Other than that, how is your experience with Faugra going?

**Comment by daniel-hritcu** [comment]
@zvictor 
On 0.0.44 is great. I manually added your patch for the global fetch since I use cloudflare workers.

I decided to install and configure graphq-generator separate since I did not manage to use faugra to generate usable code.

I'm waiting to see what's new in the next version.

Also, I'm up to help maintain this project f you want, as it will be used in a production environment for our app.

**Comment by zvictor** [comment]
awesome! I haven't heard much from users of Faugra until now, so it's great to hear that it's being useful to you 💎

**Comment by zvictor** [comment]
awesome! I haven't heard much from users of Faugra until now, so it's great to hear that it's being useful to you 💎

Please check if the new version I just published works as expected. I removed the postinstall script completely, as I believe that the newer typings declaration do the job better. I won't, however, be sure that it's working until I hear back from more people like you.

---

I have just added a [contributing](https://zvictor.github.io/faugra/#/?id=contributing) section to make it easier for newcomers to help building our project and community. You are more than welcome to join us as a maintainer!

#26 is what is bothering me in the short term, and #1 is the big beast that is stopping me from actually launching Faugra.
Any post in forums/discord of Fauna to bring their attention to #1 would be very appreciated as well!

**Comment by zvictor** [comment]
I have just released a new version (0.0.48) in which I refactored the whole exports of ESM and CJS files.
This time I added lots of tests to make sure it works better than before.

Please let me know if it breaks anything on your side.
